regex-assembly/933210.ra

```diff
@@ -0,0 +1,31 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##! Helpers
+##!> define single_or_double_quotes ['"]
+##!> define string_in_round_brackets \(.+\)
```

Comment:
I think you need to replace the any match with `[^)]`. If you have more than one pair of parentheses in the string, you might match much more than you want otherwise. For example, in `a dog (canis canis) often chases cats (felis catus)` your match would be `canis canis) often chases cats (felis catus`.

Reply:
this prevents matching the payload `(("sys")."t".("em"))("ls");`

Reply:
Makes sense. I would document that in the file.